packages/angular/src/selector/selector.component.ts

```diff
@@ -142,11 +148,14 @@ export class SelectorComponent extends RootComponent { // implements OnInit, OnC
         this.uid = _.uniqueId(`sa_${moduleName}_component_`)
         Stratus.Instances[this.uid] = this
 
+        // Declare Observable with Subscriber (Only Happens Once)
+        this.dataSub = new Observable(subscriber => this.dataDefer(subscriber))
+
         // SVG Icons
-        iconRegistry.addSvgIcon(
-            'selector:delete',
-            sanitizer.bypassSecurityTrustResourceUrl('/Api/Resource?path=@SitetheoryCoreBundle:images/icons/actionButtons/delete.svg')
-        )
+        _.forEach({
```

Comment:
This new logic properly registers all the SVGs below to their key name and stores the resulting data to the Icon Registry via XHR.